src/components/auth/AuthQR.tsx

```diff
@@ -3,7 +3,7 @@ import { EuiLoadingSpinner } from '@elastic/eui';
 import styled from 'styled-components';
 import { observer } from 'mobx-react-lite';
 import { AuthProps } from 'people/interfaces';
-import { formatRelayPerson } from 'helpers';
+import { formatRelayPerson } from 'helpers/helpers-extended';
```

Comment:
I'm confused why we need to make this change

Reply:
I will change it back!!

Reply:
I just checked why I made this change, It's throwing a typescript error on my machine.

`Module '"helpers"' has no exported member 'formatRelayPerson'.`

Reply:
thats weird cause helpers exports everything in helpers-extended can you try using the relative path instead of the shortened path?

Reply:
I figured the error was because we are importing a `.txs` file inside a `.ts` file, [here](https://github.com/stakwork/sphinx-tribes-frontend/blob/21975b94c30de5e83cafff0416e79f3e2daef751/src/helpers/index.ts#L1)